### DISCUSSION TITLE
Get diff between Github auto-generated zip

### DISCUSSION THREAD

**Discussion by ALIENQuake** [community/community#24819] [Apps, API and Webhooks]
<p>Hello,</p>
<p>It’s possible to get diff between Github auto-generated zip files?</p>
<p>Example:</p>
<p>A: <a href="https://github.com/ALIENQuake/ModA/archive/1.2.0.zip" rel="nofollow noopener">https://github.com/ALIENQuake/ModA/archive/1.2.0.zip</a></p>
<p>B: <a href="https://github.com/ALIENQuake/ModA/archive/1.1.1.zip" rel="nofollow noopener">https://github.com/ALIENQuake/ModA/archive/1.1.1.zip</a></p>
<p>both links will get me zip file, I know that it’s possible to generate diff between those two files <strong>after downoading them.</strong></p>
<p>But I wonder if it’s possible to get the diff directly?</p>

**Comment by lee-dohm** [comment]
<p>It won’t show you the exact diff in diff file format, but you can compare the two tags via the GitHub UI: <a href="https://github.com/ALIENQuake/ModA/compare/1.1.0...1.2.0" rel="nofollow noopener">https://github.com/ALIENQuake/ModA/compare/1.1.0…1.2.0</a></p>
<p>Is that the information you’re looking for?</p>

**Comment by ALIENQuake** [comment, marked as answer]
<p>No, not really. But I now know that what I was looking for is not a Github feature and my requires isn’t realistic. It would require too much processing power and storage. </p>
